workflows.tf

```diff
@@ -170,3 +170,12 @@ resource "time_sleep" "wait_after_all_workflows" {
     data.http.call_workflows_project_setup,
   ]
 }
+
+# Stop the PHS cluster after creation since it costs too much.
+data "http" "call_stop_cluster" {
```

Comment:
This looks good to me - can you confirm you tested that this worked as expected?